server/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -160,11 +160,13 @@ public Content render(Params params) {
 
   /** Renders {@code data} including the question and any existing answer to it. */
   private DivTag renderQuestionSummary(AnswerData data, Messages messages, long applicantId) {
-    DivTag questionPrompt = div(data.questionText()).withClasses("font-semibold");
-    if (!data.applicantQuestion().isOptional()) {
-      questionPrompt.with(ViewUtils.requiredQuestionIndicator());
-    }
-    DivTag questionContent = div(questionPrompt).withClasses("pr-2");
+    DivTag questionContent =
+        div(div()
+                .with(
+                    TextFormatter.formatText(
```

Comment:
This will format the question text on the applicant summary/review screen where it lists all the questions.